Tests/MapboxMapsTests/Annotations/AnnotationOrchestratorImplTests.swift

```diff
@@ -181,4 +179,195 @@ final class AnnotationOrchestratorImplTests: XCTestCase {
             }
         }
     }
+
+    func testAnnotationOrchestratorProxiesGestureDragEndForAllManagersButPoint() {
+        // given
+        let annotationManagerLayerId = "managerId"
+        let annotationManager = MockAnnotationManager()
+        factory.makePolygonAnnotationManagerStub.defaultReturnValue = annotationManager
+        factory.makePolylineAnnotationManagerStub.defaultReturnValue = annotationManager
+        factory.makeCircleAnnotationManagerStub.defaultReturnValue = annotationManager
+        longPressGestureRecognizer.getStateStub.defaultReturnValue = .ended
+        let factories: [(String, LayerPosition?) -> AnnotationManagerInternal] = [
+            impl.makeCircleAnnotationManager,
+            impl.makePolygonAnnotationManager,
+            impl.makePolylineAnnotationManager,
+        ]
+
+        for factory in factories {
+            _ = factory(annotationManagerLayerId, nil)
+        }
+
+        // when
+        longPressGestureRecognizer.sendActions()
+
+        // then
+        XCTAssertEqual(annotationManager.handleDragEndedStub.invocations.count, 1)
```

Comment:
Shouldn't here be three invocations? Since this annotation manager is simultaneously polygon, polyline and circle manager.